pkg/odo/cli/catalog/describe/service.go

```diff
@@ -6,21 +6,31 @@ import (
 	"strings"
 
 	"github.com/olekukonko/tablewriter"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
 	"github.com/openshift/odo/pkg/odo/genericclioptions"
 	svc "github.com/openshift/odo/pkg/service"
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/spf13/cobra"
 	ktemplates "k8s.io/kubectl/pkg/util/templates"
 )
 
 const serviceRecommendedCommandName = "service"
 
 var (
-	serviceExample = ktemplates.Examples(`  # Describe a service
-    %[1]s mysql-persistent`)
+	serviceExample = ktemplates.Examples(`  # Describe a service catalog service
+    %[1]s mysql-persistent
+	
+	# Describe a operator backed service
+	%[1]s 
+	`)
 
-	serviceLongDesc = ktemplates.LongDesc(`Describe a service type.
+	serviceLongDesc = ktemplates.LongDesc(`Describes a service type.
+	This command supports both service catalog services and operator backed services.
```

Comment:
Whoever reviews this PR, I am adding the documentation in long description because we still don't know where to actually add documentation.

Reply:
Makes sense.